### DISCUSSION TITLE
Click to zoom on numbered lists and headings

### DISCUSSION THREAD

**Discussion by jamie9090** [vslinko/obsidian-zoom#30] [Ideas]
Possible to add click to zoom for numbered lists too?

And any ideas if eventually we can have click to zoom on headings?

Sorry if this is the wrong place to post.

Thank you!

**Comment by vslinko** [comment]
@jamie9090 Hello, thank you for your feedback!

Numbered lists support was released in https://github.com/vslinko/obsidian-zoom/releases/tag/0.1.2
Could you please update the plugin and test it again?

Click to zoom on headers is not supported yet. I will convert your issue to a feature request.

**Comment by vslinko** [comment]
Oops, I missed that you meant click to zoom for numbered lists, not just support for numbered lists.

**Comment by chrisgrieser** [comment]
I would love to have click-to-zoom also for headings! 
(Could re-open #25, almost didn't see it being a a discussion instead of an issue.)

**Comment by fpigeonjr** [comment]
yes this would be a nice feature
